### ISSUE TITLE
Set up AppVeyor testing

### ISSUE THREAD

**Issue by yurrriq** [prometheus-erl/prometheus.erl#47]
Related: #46

For testing on Windows, I've had success in the past with AppVeyor. See [exercism/xlfe/appveyor.yml](https://github.com/exercism/xlfe/blob/master/appveyor.yml). Per #46 we'll want to set up a Windows-friendly hook for the [set-edoc-hidden script](https://github.com/deadtrickster/prometheus.erl/blob/master/bin/set-edoc-hidden) too.

**Comment by deadtrickster** [comment]
I set up AppVeyor for this repository. It's time to dust my Windows scripting foo 😄 .

**Comment by deadtrickster** [comment]
OK Windows script almost ready. Yml file is next?

**Comment by yurrriq** [comment]
Yeah, I think. We should probably not bother with `elvis` for now, but rather just run `rebar3 do xref, dialyzer, eunit`. The xlfe example should be useful in writing the `appveyor.yml`.

**Comment by NelsonVides** [comment]
Closing as it doesn't apply anymore.
